src/components/inputs/promocode-input.js

```diff
@@ -15,7 +15,7 @@ import React from 'react';
 import AsyncSelect from 'react-select/lib/Async';
 import {queryPromocodes} from '../../utils/query-actions';
 
-const PromocodeInput = ({summitId, error, value, onChange, id, multi, ...rest}) => {
+const PromocodeInput = ({summitId, error, value, onChange, id, multi, perPage, extraFilters, ...rest}) => {
```

Comment:
@romanetar pleas add default values for perPage and extraFilters using prop types